data/transactions/logic/globalfield_string.go

```diff
@@ -0,0 +1,28 @@
+// Code generated by "stringer -type=GlobalField"; DO NOT EDIT.
+
+package logic
+
+import "strconv"
+
+func _() {
+	// An "invalid array index" compiler error signifies that the constant values have changed.
+	// Re-run the stringer command to generate them again.
+	var x [1]struct{}
+	_ = x[MinTxnFee-0]
+	_ = x[MinBalance-1]
+	_ = x[MaxTxnLife-2]
+	_ = x[ZeroAddress-3]
+	_ = x[GroupSize-4]
+	_ = x[invalidGlobalField-5]
+}
```

Comment:
what kind of magic is it?

Reply:
x has one element. x[0] is the only valid index. if any of the constants change, this generated code will break signaling that it needs to be re-generated. (I'd normally not check in generated code, but we're already doing that elsewhere with this same `stringer` generator)